.github/pull_request_template.md

```diff
@@ -19,20 +19,18 @@ Closes #ISSUE
 
 ### Checklist
 
-Before creating a PR, run through this checklist and mark each as complete.
-
-- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md)
-- [ ] I have signed the [F5 Contributor License Agreement (CLA)](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md)
-- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works
-- [ ] If applicable, I have checked that any relevant tests pass after adding my changes
-- [ ] I have updated any relevant documentation ([`README.md`](/README.md) and [`CHANGELOG.md`](/CHANGELOG.md))
-- [ ] I have rebased my branch onto main
-- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork
-- [ ] If the change involves:
-  - Code
-  - Anything that resembles Personally identifying information (PII)
-    - Make sure to use placeholders such as `<username>` in place of PII
-  - URLs (watch for [typosquatting](https://support.microsoft.com/en-us/topic/what-is-typosquatting-54a18872-8459-4d47-b3e3-d84d9a362eb0))
-  - Significant new/revised content
-    
-  In these cases, the change will require at least two (2) approvals before merging
+Before merging a pull request, run through this checklist and mark each as complete.
+
+- [] I have read the [contributing guidelines](/CONTRIBUTING.md)
+- [] I have signed the [F5 Contributor License Agreement (CLA)](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md)
+- [] I have ensured that documentation content adheres to [the style guide](/templates/style-guide.md)
+- [] If the change involves potentially sensitive changes, I have assessed the possible impact
+- [] If applicable, I have added tests that prove my fix is effective or that my feature works
+- [] If applicable, I have checked that any relevant tests pass after adding my changes
+- [] I have updated any relevant documentation ([`README.md`](/README.md) and [`CHANGELOG.md`](/CHANGELOG.md))
+- [] I have rebased my branch onto main
+- [] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork
+
+Potentially sensitive changes include anything involving code, personally identify information (PII), live URLs or significant amounts of new or revised documentation.
```

Comment:
```suggestion
- [] If applicable, I have added tests that prove my fix is effective or that my feature works
- [] If applicable, I have checked that any relevant tests pass after adding my changes
- [] I have updated any relevant documentation ([`README.md`](/README.md) and [`CHANGELOG.md`](/CHANGELOG.md))
- [] I have rebased my branch onto main
- [] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

- [] If the change involves potentially sensitive changes, I have assessed the possible impact
   - These potentially sensitive changes include: 
      - Changes to code
      - Personally identifying information
      - URLs which may be affected by [typosquatting](https://en.wikipedia.org/wiki/Typosquatting)
```

Reply:
The driving effort of this PR was to fix the template, which was caused by the existed bulleted items breaking the formatting: this re-introduces the issue.

Reply:
brainstorming: the more items the list has, the less likely I am to even read it.

There is a lot of scope creep in a PR that should have been a quick fix.

Reply:
> There is a lot of scope creep in a PR that should have been a quick fix.

I disagree, but have moved it to a new issue, #89